formats/ome-converter-plugin/tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""ome converter plugin."""
```

Comment:
We should expose `convert_image` here.

Also, we should create a `batch_convert` function that receives basically the same inputs as the plugin, and just iterates over all images and converts them. Once that function is created, it should be exposed here.

Reply:
added a new batch function and is exposed in __init__.py